### ISSUE TITLE
Autocorrect & Word Prediction

### ISSUE THREAD

**Issue by ghost** [florisboard/florisboard#2698]
### Feature idea

Hey,

i think the App is lacking of autocorrect and a few Users want word prediction. Would both be possible?

**Comment by lm41** [comment]
https://github.com/florisboard/florisboard/blob/master/README.md#highlighted-features

The word suggestions aren't implemented in the current version. They are a major goal for the 0.5 milestone.

**Comment by steadfasterX** [comment]
> https://github.com/florisboard/florisboard/blob/master/README.md#highlighted-features
> 
> The word suggestions aren't implemented in the current version. They are a major goal for the 0.5 milestone.

do you know if auto correction is within "spell checking" then as well? or is there any type of auto correction planned?

**Comment by PrivacyAndSecurity** [comment]
That would be interesting to know for sure.

**Comment by patrickgold** [comment]
> do you know if auto correction is within "spell checking" then as well? or is there any type of auto correction planned?

Yes the word prediction is coupled with a simple spell checker as well. However don't expect a sophisticated grammar-parsing spell checker, it is a word based statistical one. But it will use the same model data as the predictions.

I will close this issue as word prediction stuff is already tracked and discussed by multiple other threads.
